server/services/message-action.js

```diff
@@ -7,4 +10,28 @@ async function messageAction (claim, sender) {
     console.log('error sending message', error)
   }
 }
-module.exports = messageAction
+const sqsCalculationMessageAction = async message => {
+  const {
+    sqsPaymentQueueConfig: {
```

Comment:
Could the structure and field names of the sqsPaymentQueueConfig be changed to simplify mapping values to the sendMessage service?